robot/robot.py

```diff
@@ -52,7 +53,7 @@ def __init__(
 
         configure_logging()
 
-        LOGGER.info("Initializing Hardware...")
+        LOGGER.info("Robot (v{}) Intialising...".format(__VERSION__))
```

Comment:
Typo here

Reply:
[not a typo](https://en.oxforddictionaries.com/definition/initialize)

Reply:
[I'm afraid that it is](https://en.oxforddictionaries.com/search?utf8=%E2%9C%93&filter=dictionary&query=Intialising)

Reply:
but https://en.oxforddictionaries.com/search?utf8=✓&filter=dictionary&query=Intializing

Reply:
Perhaps you mean "In**i**tializing" or "In**i**talising"?

Reply:
https://wikidiff.com/initializing/initialising settles this dispute completely.

Reply:
@Adimote @trickeydan some general feedback here -- spotting typos in ones own writing is much harder than would be expected, even when you're told that there's a typo present.

I've found it much easier to fix this sort of error when I'm told _exactly_ what the issue is and/or had the suggested change outlined.

Here, this might have looked like:
> Typo: s/Intialising/Initialising/ (add missing second "i")

Similarly when pushing back on a review comment, including the detail of _why_ you feel the comment is wrong or doesn't need to be addressed can really help.

Here this might have looked like:
> I much prefer spelling "intialising" with an "s"; [the OED notes](https://en.oxforddictionaries.com/definition/initialize) that it's a valid spelling too.

Bonus: you might enjoy adding a spellchecker to your code editors. In VSCode I use https://marketplace.visualstudio.com/items?itemName=swyphcosmo.spellchecker; I'm sure similar plugins exist for other editors.